server/lib/sync-agent.js

```diff
@@ -152,6 +165,14 @@ export default class SyncAgent {
     return this.streamQuery(query, options)
       .then(stream => this.sync(stream, started_sync_at))
       .catch(err => {
+        const { message, status } = err;
+        if (status === 400 || (err && err.routine && (err.routine.match("scanner_yyerror") || err.routine.match("parserOpenTable")))
```

Comment:
@michaloo probably yes but it would take same time to reproduce some errors and check all responses from all adapters. Should I take a closer look on it ?